cli/crates/backend/src/project.rs

```diff
@@ -181,12 +167,41 @@ pub async fn init(name: Option<&str>, template: Template<'_>) -> Result<(), Back
 
             schema_write_result?;
             dot_env_write_result?;
+
+            // This has to be the last thing we do.
+            if let ConfigType::TypeScript = config_type {
+                add_dependencies(&project_path)?;
+            };
         }
     };
 
     Ok(())
 }
 
+fn add_dependencies(project_path: &Path) -> Result<(), BackendError> {
+    environment::add_dev_dependency_to_package_json(
+        project_path,
+        GRAFBASE_SDK_PACKAGE_NAME,
+        GRAFBASE_SDK_PACKAGE_VERSION,
+    )
+    .map_err(BackendError::CommonError)?;
+
+    let package_manager = environment::guess_project_package_manager(project_path, &project_path.join("package.json"))
```

Comment:
I'd add this as a const

Reply:
Lazy value you mean?

Reply:
Just the `"package.json"` part :)

Reply:
I wrapped the whole thing to a lazy cell, because we call this more than once and it makes sense to cache the result.